packages/design-system/src/components/Tabs/Tabs.test.tsx

```diff
@@ -53,6 +53,21 @@ describe('Tabs', function () {
     expect(firstPanel.getAttribute('aria-labelledby')).toEqual(firstTab.id);
   });
 
+  it('should render a default aria-label when none is provided', () => {
```

Comment:
I don't think we want this to render a default value when nothing is provided, because it is possible that another element could label the tablist. In that case, a user would want to use the `aria-labelledby` attribute and omit the `aria-label`. A good example of this use case is [here](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/#ex_label). You can see in the source code that the h3 element is labelling the tablist and that an aria label is unnecessary.

Reply:
@jack-ryan-nava-pbc Thanks for pointing this out! I’ve removed the default value, so aria-label will only be applied if explicitly provided via the prop. Do you think our [accessibility guidance for the Tabs component](https://design.cms.gov/components/tabs/?theme=core#accessibility) in the documentation could use a bit more nuance?

Reply:
I think we should update the accessibility guidance for the Tabs component, maybe to highlight that the aria-label is  optional if another element is describing the content via `aria-describedby`. But we can revisit this in another ticket.